src/GitVersionTfsTask/task.json

```diff
@@ -35,6 +35,15 @@
       "helpMarkDown": "Whether to update versions in the AssemblyInfo files"
     },
     {
+      "name": "updateAssemblyInfoFilename",
+      "type": "string",
+      "label": "Update Assembly File",
+      "defaultValue": "",
+      "required": false,
+      "helpMarkDown": "Update versions in specified file",
```

Comment:
Agreed, it doesn't support wildcard right now (but that's a good idea and something that would be easy to do I guess, does the GitVersion parameter support multiple files ?)